### DISCUSSION TITLE
AI第一股 商湯首日漲7%

### DISCUSSION THREAD

**Discussion by kaoyihua27** [1101-GS3073-C/Python-Community-GS3073-C#172] [General]
因為美國二度封殺而延後上市的AI第一股商湯科技，昨（30）日在香港上市，首日股價表現不錯，盤中由掛牌價港幣3.91元一度拉升逾兩成，最終收盤價港幣4.13元，上漲7.27%。
https://udn.com/news/story/7333/6000051

**Comment by pocper1** [comment]
AI 是日後熱門項目，感覺有閒錢可以入股了？

**Comment by cobb110202008** [comment]
感覺以後AI相關產業會蓬勃發展，進而股價也跟著上漲。

**Comment by briankuo123** [comment]
等技術越來越成熟之後，有越多人看好AI的發展，到時候AI相關公司的股票應該都會很夯。

**Comment by jim11261314** [comment]
我覺得還是要有實際的利潤，不然現在能應用並能取代已有產業的能力還太低
可能只是一個炒作的題材

**Comment by 110409538** [comment]
我認為AI在未來的潛力非常大。

**Comment by Yunhans** [comment]
AI也可以有股票喔

**Comment by LanningYang** [comment]
個人覺得未來AI一定有更好的成長，但近期應該還不會如預期的發展
